e2e/run.sh

```diff
@@ -12,23 +12,31 @@
 
 set -e
 
-cd ..
-cd docker
-./run.sh
-./waitForDocker.sh
-cd ..
-cd e2e
+for APP_SERVER in tomcat jetty; do
+  export APP_SERVER
+ 
+  cd ..
+  cd docker
+  ./run.sh
+  ./waitForDocker.sh
+  cd ..
+  cd e2e
 
-sleep 10
+  sleep 10
 
-if [ ! -d  'node_modules' ]; then
-  echo "npm ci"
-  npm ci
-fi
+  if [ ! -d  'node_modules' ]; then
+    echo "npm ci"
+    npm ci
+  fi
 
-docker ps
+  docker ps
 
-npx playwright install --with-deps # install browsers
-npx playwright test
-exit $?
+  npx playwright install --with-deps # install browsers
+  npx playwright test
+
+  cd ..
+  cd docker
+  ./shutdown.sh 
+done
 
+exit $?
```

Comment:
What happens if the test fail for tomcat but not jetty? Does it still error out?

Reply:
hmz yes, that's on the TODO list  (and it probably doesn't hurt to update the docker documentation as well...)

Reply:
fixed